server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -2164,46 +2167,56 @@ public void doInTransactionWithoutResult(final TransactionStatus status) {
     @Override
     @ActionEvent(eventType = EventTypes.EVENT_POD_CREATE, eventDescription = "creating pod", async = false)
```

Comment:
@shwstppr can you disect this method in smaller parts?

Reply:
done